File: SpringBootDocker/src/main/java/com/example/java/SimpleController.java
package com.example.java;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/")
public class SimpleController {

    @GetMapping
    public String message() {
        return "Hello Jib With Java!!";
    }
}

File: SpringBootDocker/src/main/kotlin/com/example/kotlin/SimpleController.kt
package com.example.kotlin

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/")
class SimpleController {

    @GetMapping
    fun message(): String {
        return "Hello Jib With Kotlin!!"
    }
}
